pandas/core/arrays/integer.py

```diff
@@ -501,23 +501,6 @@ def value_counts(self, dropna=True):
 
         return Series(array, index=index)
 
-    def _values_for_argsort(self) -> np.ndarray:
```

Comment:
does we have a benchmark for sorting Int64? so does this change things?

Reply:
I can't found the benchmark for `argsort`. But this change slows down the performance.
HEAD:
```
In [2]: arr = pd.array([3, None, 1], dtype='Int64')                             

In [3]: %timeit arr.argsort()                                                   
28.6 µs ± 190 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

master:
```
In [2]: arr = pd.array([3, None, 1], dtype='Int64')                             

In [3]: %timeit arr.argsort()                                                   
5.73 µs ± 15.9 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

```

Reply:
can you add this benchmark (make the arrays longer though)